x-pack/test/apm_api_integration/tests/inspect/inspect.ts

```diff
@@ -49,7 +49,7 @@ export default function customLinksTests({ getService }: FtrProviderContext) {
             },
           });
           expect(status).to.be(200);
-          expect(body._inspect?.length).to.be(2);
+          expect(body._inspect?.length).to.be(1);
```

Comment:
I assume the api test changes are exclusively caused by the change from metrics to transactions?

Reply:
yes